client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
It does get rendered as `data-tip-target`, and the intention was to use something that other components could use as well. Anything we can do to make this clearer?